src/lib.rs

```diff
@@ -301,28 +299,26 @@ impl Ngram {
 // to a constructor method, and allowing default values by omission.
 #[derive(Debug, Default)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct NgramBuilder {
-    arity: usize,
+pub struct NgramBuilder<const ARITY: usize> {
     pad_left: Pad,
     pad_right: Pad,
     text: String,
 }
 
-impl NgramBuilder {
+impl<const ARITY: usize> NgramBuilder<ARITY> {
     /// Initialize a new instance of an `NgramBuilder`, with a default `arity`
     /// of 2, padding set to `Auto`, for the given `text`.
```

Comment:
This comment is no longer valid - the ARITY generic has no default, and as I understand it const generics still don't have the capability to be defaulted, so I think this doc comment needs a correction.